Automattic/simplenote-macos
# Update Tracks to send `userID` Field.

Swapping username argument with userID so Tracks events will still be recorded (for users that have opted in).

**To Test**
Enable analytics in the app settings, and open or create a note. You should see the event show in the live view in Tracks after 5 minutes or so, and it should have the userid field populated.
